invenio_files_rest/config.py

```diff
@@ -22,12 +22,21 @@
 # waive the privileges and immunities granted to it by virtue of its status
 # as an Intergovernmental Organization or submit itself to any jurisdiction.
 
-"""Invenio File Rest module configuration file."""
+"""Invenio Files Rest module configuration file."""
 
 FILES_REST_STORAGE_CLASS_LIST = {
```

Comment:
The idea behind storage class is to allow a system to record what type of storage a given file resides on - e.g. offline/online, reduced resilience, ... . In Files-REST there's no specific functionality associated with it, but an Invenio instance may use it to adapt behaviour depending no the storage class.